### PR TITLE
Fix CVE-2022-42003 and CVE-2022-42004

### PR DESCRIPTION
**Pull request by brfrn169**
This PR fixes CVE-2022-42003 and CVE-2022-42004 by upgrading the Azure Cosmos client. Please take a look!

### CODE REVIEW

File: core/build.gradle [hunk 1]
```diff
@@ -142,7 +142,7 @@ task integrationTestCosmos(type: Test) {
         systemProperties(System.getProperties())
     }
     maxParallelForks = 3
-    jvmArgs '-XX:MaxDirectMemorySize=2g'
+    jvmArgs '-XX:MaxDirectMemorySize=2g', '-Xmx4g'
```
Comment (brfrn169) [inline core/build.gradle:145]: OOM error occurred in the Cosmos Integration test after this change. So increased the heap size.

**Review by Torch3333 — APPROVED** [review]
LGTM, thank you!

**Review by komamitsu — APPROVED** [review]
Looks good! 👍

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**